ospfd/ospf_vty.c

```diff
@@ -8208,6 +8211,9 @@ static int ospf_vty_dead_interval_set(struct vty *vty, const char *interval_str,
 				ospf_nbr_timer_update(oi);
 	}
 
+	if (set_fast_hello) 
```

Comment:
Isn't this the same as checking if `params->fast_hello != OSPF_FAST_HELLO_DEFAULT`?